src/app/pages/sign-out-confirm/sign-out-confirm.layout.tsx

```diff
@@ -40,7 +40,7 @@ export function SignOutConfirmLayout(props: SignOutConfirmLayoutProps) {
               ledger: ` you'll need to reconnect your Ledger to sign back into your wallet.`,
             })}
           </styled.p>
-          <styled.p mt="space.05" textStyle="label.02">
+          <styled.div mt="space.05" textStyle="label.02">
```

Comment:
I changed this as to `<styled.div` as `<Flag` is a `<div` and not a valid descendant of `<p`